L1Trigger/CSCTriggerPrimitives/python/cscTriggerPrimitiveDigis_cfi.py

```diff
@@ -508,6 +508,18 @@
                            )
 
 ## GEM-CSC ILT in ME1/1
+from Configuration.Eras.Modifier_run2_GEM_2017_MCTest_cff import run2_GEM_2017_MCTest
+run2_GEM_2017_MCTest.toModify( cscTriggerPrimitiveDigis,
```

Comment:
@slava77 I don't think so. From the UCLA colleagues I got the message that they plan to run some tests with the VFAT2 indeed. However, it will not be possible to develop the new LCT algorithm in firmware since there is a mistake in the physical mapping of strips to the VFAT2 channels in the hybrid boards. They want to wait until the prototype VFAT3 is out.